# HG changeset patch
# User jwe@segfault.lan
# Date 1201926224 18000
#      Fri Feb 01 23:23:44 2008 -0500
# Node ID 710aea220a2c1477718b65855759c72c169cb875
# Parent  fc55a5e1760b832a672d9de3b53888a3d6793d21
Added tag ss-2-9-14 for changeset fc55a5e1760b

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -24,8 +24,9 @@ b2e1be30c8e94902e6557c1edba2c4f37ff7bdf8
 d542d919783961dce51ee8fc06f65b8f2cfe4156 ss-2-9-8
 60f9ced8ab534e65f27d95a712d97ba6add6cd8d ss-2-9-9
 12b676a0b18395af2fa0ccceb516447d8be6ad03 before-graphics-branch
 a848b846cb3ac279782b7fef8270a62ca4578539 ss-2-9-10
 3e2cfed6c87e2019cb1584211068b9c581331ccb before-object-branch
 10da0f6d85c260bb5f640436e42d633a57ccda9b ss-2-9-11
 db6d908e8a03a42760a45c7fd29eae074e505f7c ss-2-9-12
 afbd31bb7b4e001c6c16d41eb908aa66bed64352 ss-2-9-13
+fc55a5e1760b832a672d9de3b53888a3d6793d21 ss-2-9-14
